coralproject/talk
# [CORL-986] Perspective Bugs

## What does this PR do?

Sometimes when the API updates the perspective config, the endpoint is set to null. This fixes that by guarding.



## What changes to the GraphQL/Database Schema does this PR introduce?



None.

## How do I test this PR?



Try to set the `endpoint` to null via the CLI, note that when a comment is processed by perspective it does not fail.
